projects/1.18/assets/biome-makeover-forge/biomemakeover/lang/zh_cn.json

```
@@ -0,0 +1,568 @@
+  "block.biomemakeover.white_terracotta_brick_stairs" : "品红色陶瓦砖楼梯",
+
+  "block.biomemakeover.glowshroom_stem" : "荧光菇柄",
+  "block.biomemakeover.mycelium_sprouts" : "菌丝苗",
+  "block.biomemakeover.tall_brown_mushroom" : "高棕色蘑菇",
+  "block.biomemakeover.tall_red_mushroom" : "高红色蘑菇",
+  "block.biomemakeover.potted_mycelium_roots" : "菌丝菌索盆栽",
+  "block.biomemakeover.potted_purple_glowshroom" : "紫色荧光菇盆栽",
+  "block.biomemakeover.potted_green_glowshroom" : "绿色荧光菇盆栽",
+  "block.biomemakeover.potted_orange_glowshroom" : "橙色荧光菇盆栽",
+  "block.biomemakeover.potted_blighted_balsa_sapling" : "染菌轻木树苗盆栽",
+  "block.biomemakeover.mycelium_roots" : "菌丝菌索",
+  "block.biomemakeover.blighted_balsa_log" : "染菌轻木原木",
+  "block.biomemakeover.stripped_blighted_balsa_log" : "去皮染菌轻木原木",
+  "block.biomemakeover.stripped_blighted_balsa_wood" : "去皮染菌轻木",
+  "block.biomemakeover.blighted_balsa_wood" : "染菌轻木",
+  "block.biomemakeover.blighted_balsa_planks" : "染菌轻木木板",
+  "block.biomemakeover.blighted_balsa_leaves" : "染菌轻木树叶",
+  "block.biomemakeover.blighted_balsa_sapling" : "染菌轻木树苗",
+  "block.biomemakeover.blighted_balsa_pressure_plate" : "染菌轻木压力板",
+  "block.biomemakeover.blighted_balsa_fence" : "染菌轻木栅栏",
+  "block.biomemakeover.blighted_balsa_fence_gate" : "染菌轻木栅栏门",
+  "block.biomemakeover.blighted_balsa_button" : "染菌轻木按钮",
+  "block.biomemakeover.blighted_balsa_trapdoor" : "染菌轻木活板门",
+  "block.biomemakeover.blighted_balsa_door" : "染菌轻木门",
+  "block.biomemakeover.blighted_balsa_stairs" : "染菌轻木楼梯",
+  "block.biomemakeover.blighted_balsa_slab" : "染菌轻木台阶",
+  "block.biomemakeover.blighted_balsa_sign" : "染菌轻木告示牌",
+  "block.biomemakeover.blighted_balsa_wall_sign" : "染菌轻木告示牌",
+  "block.biomemakeover.willow_log" : "柳木原木",
+  "block.biomemakeover.stripped_willow_log" : "去皮柳木原木",
+  "block.biomemakeover.stripped_willow_wood" : "去皮柳木",
+  "block.biomemakeover.willow_wood" : "柳木",
+  "block.biomemakeover.willow_planks" : "柳木木板",
+  "block.biomemakeover.willow_leaves" : "柳树树叶",
+  "block.biomemakeover.willow_sapling" : "柳树树苗",
+  "block.biomemakeover.willow_pressure_plate" : "柳木压力板",
+  "block.biomemakeover.willow_fence" : "柳木栅栏",
+  "block.biomemakeover.willow_fence_gate" : "柳木栅栏门",
+  "block.biomemakeover.willow_button" : "柳木按钮",
+  "block.biomemakeover.willow_trapdoor" : "柳木活板门",
+  "block.biomemakeover.willow_door" : "柳木门",
+  "block.biomemakeover.willow_stairs" : "柳木楼梯",
+  "block.biomemakeover.willow_slab" : "柳木台阶",
+  "block.biomemakeover.willow_sign" : "柳木告示牌",
+  "block.biomemakeover.willow_wall_sign" : "柳木告示牌",
+  "block.biomemakeover.swamp_cypress_wood" : "沼泽柏木",
+  "block.biomemakeover.swamp_cypress_log" : "沼泽柏原木",
+  "block.biomemakeover.stripped_swamp_cypress_wood" : "去皮沼泽柏木",
+  "block.biomemakeover.stripped_swamp_cypress_log" : "去皮沼泽柏原木",
+  "block.biomemakeover.swamp_cypress_planks" : "沼泽柏木板",
+  "block.biomemakeover.swamp_cypress_leaves" : "沼泽柏树叶",
+  "block.biomemakeover.swamp_cypress_sapling" : "沼泽柏树苗",

```

```suggestion
  "block.biomemakeover.swamp_cypress_log" : "沼泽柏木原木",
  "block.biomemakeover.stripped_swamp_cypress_wood" : "去皮沼泽柏木",
  "block.biomemakeover.stripped_swamp_cypress_log" : "去皮沼泽柏木原木",
  "block.biomemakeover.swamp_cypress_planks" : "沼泽柏木木板",
  "block.biomemakeover.swamp_cypress_leaves" : "沼泽柏树树叶",
  "block.biomemakeover.swamp_cypress_sapling" : "沼泽柏树树苗",
```
不知道是否要像`深色橡木`一样处理。
此外，`柏木`？`柏树`？